frontend/src/libs/components/table/table.tsx

```diff
@@ -74,23 +80,40 @@ const Table = <T,>({
     [changePageSize, table, changePageIndex],
   );
 
+  if (data.length === 0 && !isLoading) {
+    return emptyTableMessage ? (
+      <div className={getValidClassNames('h4', styles.message)}>
+        There are no data here yet. Please,{' '}
+        <span className={styles.red}>{emptyTableMessage}</span>
```

Comment:
I don't like the idea of passing only a part of the message for placeholder

Reply:
In case we pass the full message we can’t set two colors for this message how on the layout

Reply:
I agree with @kalinkaaaa14, It's quite confusing to pass only part of the text. You can pass JSX if you need to substitute default message